src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+import { TodoCondition } from '../../types/TodoCondition';
+
+type Props = {
+  todos: Todo[],
+  onDeleteTodo: (todoId: number) => void,
+  procesingTodosId: number[],
+  todoCondition: TodoCondition,
+  toggleTodo: (curentTodo: Todo[], status?: boolean | undefined) => void,
```

Comment:
Change the type of 'status' from 'boolean | undefined' to 'boolean?' to simplify the type definition.